### DISCUSSION TITLE
[Feature] Manual Face Recognition

### DISCUSSION THREAD

**Discussion by XCoolBeatX** [immich-app/immich#8878] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Hi All,

As far as I understand this feature is not present in the Immich and no one has discussed the idea of this feature.

At the moment Immich has a good Facial Recognition capability however sometimes it does miss certain people. It would be cool to have a similar feature that allows you to add people manually. Something similar to Facebook's mechanism would be amazing or the ability to draw a square around their head and then name them.

I look forward to your comments on the idea

### Platform

- [ ] Server
- [X] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
#7301
